File: src/main/java/uk/gov/justice/hmpps/casenotes/OffenderCaseNotesApplication.java
package uk.gov.justice.hmpps.casenotes;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.boot.context.properties.EnableConfigurationProperties;
import org.springframework.security.oauth2.config.annotation.web.configuration.EnableResourceServer;
import uk.gov.justice.hmpps.casenotes.config.OffenderCaseNoteProperties;

@SpringBootApplication
@EnableConfigurationProperties(OffenderCaseNoteProperties.class)
@EnableResourceServer
public class OffenderCaseNotesApplication {
    public static void main(final String[] args) {
        SpringApplication.run(OffenderCaseNotesApplication.class, args);
    }

}

File: src/main/java/uk/gov/justice/hmpps/casenotes/OffenderCaseNotesApplication.kt
package uk.gov.justice.hmpps.casenotes

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.context.properties.EnableConfigurationProperties
import org.springframework.boot.runApplication
import org.springframework.scheduling.annotation.EnableAsync
import uk.gov.justice.hmpps.casenotes.config.ServiceConfig

@SpringBootApplication
@EnableAsync
@EnableConfigurationProperties(ServiceConfig::class)
class OffenderCaseNotesApplication

fun main(args: Array<String>) {
  runApplication<OffenderCaseNotesApplication>(*args)
}
